src/components/Issues.js

```diff
@@ -1,42 +1,97 @@
 import styled from 'styled-components/macro';
 import pin from '../images/pin.svg';
+import pinFill from '../images/pin-fill.svg';
 
-function Issues({ issues }) {
+export default function Issues({ issues, togglePin }) {
   return (
     <>
-      {issues.map(issue => (
-        <Wrapper key={issue.id} title={issue.title} state={issue.state}>
-          <Title>{issue.title}</Title>
-          <State>{issue.state}</State>
-        </Wrapper>
-      ))}
+      <HeadingWrapper>
+        <HeadingTitle>Title</HeadingTitle>
+        <HeadingState>State</HeadingState>
+      </HeadingWrapper>
+
+      {issues.map(issue => {
+        const isPinned = issues.find(
+          issueId => issueId.id === issue.id
+        ).clicked;
```

Comment:
this part I do not understand, but it seems to be right :)

Reply:
const isPinned will find the current id and extract the actual state (clicked: true / false ) from my fetched array (issues).